tests/conftest.py

```diff
@@ -360,3 +479,35 @@ def _load_yaml(yaml_in: str) -> PipelineConfig:
         return conf[0]
 
     return _load_yaml
+
+
+def _change_value_parameters_method_pipeline(
+    yaml_path: str,
+    method: list,
+    key: list,
+    value: list,
+    save_result: Union[None, bool] = None,
+):
+    # changes methods parameters in the given pipeline and re-save the pipeline
+    with open(yaml_path, "r") as f:
+        conf = list(yaml.load_all(f, Loader=yaml.FullLoader))
+    opened_yaml = conf[0]
+    methods_no = len(opened_yaml)
+    methods_no_correct = len(method)
+    for i in range(methods_no):
+        method_content = opened_yaml[i]
+        method_name = method_content["method"]
+        for j in range(methods_no_correct):
+            if method[j] == method_name:
+                # change something in parameters here
+                opened_yaml[i]["parameters"][key[j]] = value[j]
+                if save_result is not None:
+                    # add save_result to the list of keys
+                    opened_yaml[i]["save_result"] = save_result
+
+    with open(yaml_path, "w") as file_descriptor:
+        yaml.dump(
+            opened_yaml, file_descriptor, default_flow_style=False, sort_keys=False
+        )
+
+    return 0
```

Comment:
Any reason why this returns 0, none of the uses of function seem to use the return value?

Reply:
yeah, may be I had something in mind initially :) I'll remove it